src/wallet/wallet.cpp

```diff
@@ -2215,6 +2215,26 @@ void CWallet::AvailableCoins(std::vector<COutput> &vCoins, bool fOnlySafe, const
                     continue;
                 }
 
+
+                if (coinControl) {
+                    std::vector<std::vector<unsigned char> > vSolutions;
+                    txnouttype whichType;
```

Comment:
Maybe it would be saner to set a default value in case `Solver()` returns false and hasn't set `whichType`?